hw/ip/rstmgr/doc/theory_of_operation.md

```diff
@@ -151,6 +151,12 @@ Parameter                   | Default       | Description
 
 * [Interface Tables](../../../top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson#interfaces)
 
+## Reset Indications for Alert Handler
+
+The alert handler needs to know the status of the various reset domains in the system so that the ping mechanism does not produce false alert indications.
+To that end, the reset manager outputs a 4bit MuBi signal for each reset domain that indicates whether it is in reset or not.
```

Comment:
```suggestion
To that end, the reset manager outputs a 4bit MuBi signal for each reset domain that indicates whether its reset is active.
```

Or perhaps we can be more specific and say whether it can respond to ping requests.

Reply:
done